lib/auth-client.js

```diff
@@ -202,4 +202,25 @@ AuthClient.prototype.exchangeRefreshToken = function(token) {
 
 };
 
+/**
+ * Determine whether a vehicle is compatible with Smartcar
+ *
+ * @param {String} vin - the VIN of the vehicle
+ * @return {Promise.<Boolean>} true if the vehicle is compatible
+ * with Smartcar
+ */
+AuthClient.prototype.compatibility = function(vin) {
+
+  const qs = {vin};
+
+  return util.wrap(this.request.get({
+    baseUrl: config.api,
+    url: `${config.version}/compatibility`,
```

Comment:
missing `v` before version, code does not function as is :grimacing: 

```suggestion
    url: `v${config.version}/compatibility`,
```